release-content/0.15/release-notes/_release-notes.toml

```diff
@@ -126,7 +126,7 @@ file_name = "14284_Add_custom_cursors.md"
 [[release_notes]]
 title = "reflect: implement the unique reflect rfc"
 authors = ["@soqb"]
-contributors = ["@alice-i-cecile", "@MrGVSV"]
+contributors = ["@nicopap", "@alice-i-cecile", "@MrGVSV"]
```

Comment:
Added because this entire feature was largely based on their [proposal](https://github.com/bevyengine/rfcs/pull/56).

Reply:
Maybe worth boosting to an "author" credit then? `contributors` is for tweaks / minor changes to existing functionality / won't show up in the release notes, whereas `author` is for "substantial contributions worth publicly crediting". If they defined the architecture, that seems to fit the "author" bill.

Reply:
Will do!